lib/console.js

```diff
@@ -33,7 +35,8 @@ function Console(stdout, stderr) {
 }
 
 Console.prototype.log = function() {
-  this._stdout.write(util.format.apply(this, arguments) + '\n');
+  this._stdout.write(util.format.apply(this, arguments).replace(/^/gm,
+          Array(this._group * 2 + 1).join(' ')) + '\n');
```

Comment:
I just wanted to keep `console` as lightweight and performant as possible.

`String.prototype.repeat` is bit slower than `Array.join` method I chose.

See http://jsperf.com/faster-string-repeat/15

I do agree that `' '.repeat()` is more readable in code.